### DISCUSSION TITLE
Contributions - Good First Issue

### DISCUSSION THREAD

**Discussion by josh-ramer** [NVIDIA/cutlass#1535] [Q&A] [answered]
I was watching [Aniket's presentation at the Linux Foundation Open Source Summit](https://youtu.be/PWWOGrLZtZg) given ~1yr ago & then, interested in the work & wanting to learn & maybe even contribute at some point, I headed over to this repository. I see that only 2 issues have ever had the `Good First Issue` tag. Is that just not in use here? Are you welcoming contributions from anyone? Can I pick up any issue I find interesting?

**Comment by thakkarV** [comment, marked as answer]
We don't really use the good first issue tag. But of course, contributions from the community are very much welcome! What kind of GPUs do you have access to?

**Reply by josh-ramer** [reply]
I use AWS EC2 spot instances which have any of L4, Tesla M60, A10G, T4G, K80s, Tesla V100, A100, & H100 GPUs. Of course some of those can get expensive even at SPOT prices, e.g. the P5 instances with the H100s are still ~58$/hr at spot prices (currently unused compute prices). I mostly use the L4s @~9 cents/hr & my cost per day is ~1$ for a long day. If I'm being careful, I usually wouldn't reach that. I do frequently also write CPP files in Google Colab blocks & output them to disk & run them for free. I think the default instance is using a T4. 

Will that do? Do you have other alternatives I mind? Any other advice, pointers?

Thanks for the quick reply.

**Reply by thakkarV** [reply]
being able to rent out GPUs is great, but not a requirement at all for contributing. Even if you have an RTX in your local machine or no GPU at all, you should be able to do a lot. Is there specific things you are interested in working on?

**Reply by josh-ramer** [reply]
Anything involving GPU kernels is interesting to me. I'm going through the [CUTLASS docs](https://github.com/NVIDIA/cutlass/blob/main/media/docs/gemm_api_3x.md#cutlass-30-gemm-api) & [recommended CuTe tutorial & reading](https://github.com/NVIDIA/cutlass/blob/main/media/docs/cute/00_quickstart.md#tutorial) to familiarize myself. I appreciate you taking the time to help get me started Vijay, this is an incredible project that you & the team have going.

**Reply by hwu36** [reply]
i am slowly adding more `Good First Issue` now.  We welcome contributions!!
